docs/src/gensim_theme/layout.html

```diff
@@ -47,21 +55,26 @@
 
     <link rel="shortcut icon" href="{{ pathto('_static/favicon.ico', 1) }}"/>
 
-    <script type="text/javascript">
-      var _gaq = _gaq || [];
-      _gaq.push(['_setAccount', 'UA-24066335-1']);
-      _gaq.push(['_trackPageview']);
-
-      (function() {
-      var ga = document.createElement('script'); ga.type = 'text/javascript'; ga.async = true;
-      ga.src = ('https:' == document.location.protocol ? 'https://ssl' : 'http://www') + '.google-analytics.com/ga.js';
-      var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(ga, s);
-      })();
+    <script type="text/javascript">  // only commented for the moment - JD-20170830
```

Comment:
It's better to remove code than comment it out.

The history is stored, versioned and archived by git anyway, in case we ever need to go back. Like this we only bloat up the pages for no reason (slower page download, harder to read source).